src/libOpenImageIO/maketexture.cpp

```diff
@@ -931,7 +1034,20 @@ make_texture_impl (ImageBufAlgo::MakeTextureMode mode,
         mode = ImageBufAlgo::MakeTxEnvLatl;
         src = latlong;
     }
-
+    
+    if (mode == ImageBufAlgo::MakeTxBumpWithSlopes ){
+        ImageSpec newspec = src->spec();
+        newspec.width = newspec.full_width = src->spec().width;
+        newspec.height = newspec.full_height = src->spec().height;
+        newspec.tile_width = newspec.tile_height = 0;
+        newspec.format = TypeDesc::FLOAT;
+        newspec.nchannels = 6;        
```

Comment:
I think in this spot you should adjust newspec.channelnames, so that at least for exr files, the channels will have meaningful names.